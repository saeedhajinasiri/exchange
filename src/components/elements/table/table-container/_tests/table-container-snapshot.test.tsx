import React from "react";
import {cleanup, render} from "test";
import Text from '../../../text'
import TableContainer from '../table-container'
import Table from '../../table'

afterEach(cleanup)

describe('Test TableContainer UI', () => {
  const children = <Table/>
  it('Default', () => {
    const {asFragment} = render(<TableContainer>{children}</TableContainer>);
    expect(asFragment()).toMatchSnapshot();
  });

  it('Test with children prop', () => {
    const {asFragment} = render(<TableContainer>{children}</TableContainer>);
    expect(asFragment()).toMatchSnapshot();
  });

  it('Test with className prop', () => {
    const {asFragment} = render(<TableContainer className={"test-class-name"}>{children}</TableContainer>);
    expect(asFragment()).toMatchSnapshot();
  });
});
