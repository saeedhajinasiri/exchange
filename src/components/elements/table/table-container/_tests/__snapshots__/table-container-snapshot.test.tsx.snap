// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test TableContainer UI Default 1`] = `
<DocumentFragment>
  <div
    class="overflow-scroll w-full"
  >
    <table
      class="w-full"
    />
  </div>
</DocumentFragment>
`;

exports[`Test TableContainer UI Test with children prop 1`] = `
<DocumentFragment>
  <div
    class="overflow-scroll w-full"
  >
    <table
      class="w-full"
    />
  </div>
</DocumentFragment>
`;

exports[`Test TableContainer UI Test with className prop 1`] = `
<DocumentFragment>
  <div
    class="test-class-name overflow-scroll w-full"
  >
    <table
      class="w-full"
    />
  </div>
</DocumentFragment>
`;
